src/raw/document.rs

```diff
@@ -577,10 +594,10 @@ impl TryFrom<&RawDocument> for crate::Document {
 }
 
 impl<'a> IntoIterator for &'a RawDocument {
-    type IntoIter = Iter<'a>;
+    type IntoIter = Box<dyn Iterator<Item = Self::Item> + 'a>;
```

Comment:
Unfortunately, this is a breaking change.  I think this can be solved by what you suggested in the conversation - keep the existing functionality named `Iter` and call the new one `RawIter`.

Reply:
alas, easy enough done